editor/src/templates/editor.tsx

```diff
@@ -194,6 +198,16 @@ export class Editor {
 
         const projectId = getProjectID()
         if (projectId == null) {
+          // Check if this is a github import
+          const urlParams = new URLSearchParams(window.location.search)
+          const githubOwner = urlParams.get('github_owner')
+          const githubRepo = urlParams.get('github_repo')
+          if (githubOwner != null && githubRepo != null) {
+            // Trigger the repo download but do nothing with it
+            downloadGithubRepo(githubOwner, githubRepo).then((project) => {
```

Comment:
Does this mean the editor will just show our regular loading animation while this is happening? I'm thinking of large repos and GH going slow.

Likely also for the follow on work we should handle errors from this promise too.

Reply:
For this pass it doesn't block the editor and then the result is binned anyway. The next PR will address actually doing something with the editor, and then the loading animation and errors are to be tackled in #495 and #509

Reply:
*nods*